docs/src/main/asciidoc/security-jdbc.adoc

```diff
@@ -184,7 +184,8 @@ INSERT INTO test_user (id, username, password, role) VALUES (2, 'user','user', '
 [NOTE]
 ====
 It is probably useless, but we kindly remind you that you must not store clear-text passwords in production environment ;-).
-The `elytron-security-jdbc` offers a built-in bcrypt password mapper. You can see the section `Define the user entity` for more details at xref:security-getting-started-tutorial.adoc#define-the-user-entity.
+The `elytron-security-jdbc` offers a built-in bcrypt password mapper.
```

Comment:
> The `elytron-security-jdbc` extension offers a built-in bcrypt password mapper.

would be better (i.e. adding `extension`)

Reply:
I didn't change this part, but sure, truth be told when I was reviewing #37771, it took me while to realize that `elytron-security-jdbc` is actually `quarkus-elytron-security-jdbc` artifact and not something elytron specific. I'll create separate PR, I think docs PRs don't eat that much CI resources.